### ISSUE TITLE
clipboard box is always empty

### ISSUE THREAD

**Issue by Michel73** [technet/brackets.clipbox#2]
I'm using brackets 0.44 and the clipbox 0.2.1 extension on ubuntu 14.04 within a virtual machine.

**Comment by technet** [comment]
Hi I need bit more information, did you first copy some text using <kbd>Ctrl</kbd>+<kbd>C</kbd>
Then only it starts appear in the quick view drop down. There are some limitations that you can find [here](https://github.com/technet/brackets.clipbox#brackets-clipbox-clipboard-history)

Can you also launch debug mode and see chrome developer tools console.log for any exceptions, thanks

**Comment by Michel73** [comment]
Yes, first I had copied some text snippets via crtl+c but the clipbox that appears via ctrl+alt+v is always empty.
There is nothing in the console.log after I did the steps above.

**Comment by Wikunia** [comment]
Same on my machine (Arch Linux). Actually it worked for a short time but then stopped and I have no clue why. I will try to debug it within the next days.

**Comment by pdcurtis** [comment]
Same problem with Linux Mint 17.2 (Ubuntu 14.04 based)
